### ISSUE TITLE
[Bug] Privacy badger installed? Or not?

### ISSUE THREAD

**Issue by madb1lly** [mozilla-mobile/fenix#9096]
## Steps to reproduce
I don't know the steps to reproduce, because I can't reproduce it. However, this is what happened:
1. Fenix Nightly before update this morning - no tab about installation of privacy badger was present.
2. Updated to newest version via the Play store.
3. Then I have a tab which says "thank you for installing privacy badger" or something similar. The address for this tab is mozextension://... I think.

Unfortunately at this point I read and closed the tab because it didn't require action. I then went to the addons page in settings and saw that actually privacy badger wasn't there at all - it was still in my list of unsupported extensions.

So, did it install or not? Are you using my instance as a test subject? ;)

### Expected behavior
When I get a tab which says that an addon has been installed then that addon should actually be installed.

### Actual behavior
It appears that Privacy Badger has not been installed.

### Device information

* Android device: Nokia 8 on Android 9
* Fenix version: Nightly 200310 18:01 (Build #201572827) 36.0.0 b3ee6717d

**Comment by sv-sdeiac** [comment]
Hi @madb1lly, I was able to reproduce your situation using the following steps, please confirm if you can reproduce using my steps.

### Steps to reproduce
1. Download the Nightly apk build from: http://archive.mozilla.org/pub/mobile/nightly/2020/01/2020-01-20-18-04-35-mozilla-esr68-android-api-16/;
2. Open Firefox Nightly;
3. Go to add-ons and install privacy badger;
4. Update Nightly using Play Store;
5. Open Firefox Nightly;
6. On the displayed page tap "Start Firefox Nightly";
7. Scroll to the bottom of the onboarding screen and select "Start browsing";
8. Tap on the back button to view the Homepage of the App;

You should see an open tab about the Privacy badger add-on with the URL: moz-extension...
Also, if you open the settings from 3dot menu and select the Add-on option, the Privacy badger is displayed on the "Not yet supported" list.
However, it is important to note that this is not an actual issue because that add-on is not yet supported in the Firefox Preview Nightly, so when you migrate the app from Firefox Nightly to Firefox Preview Nightly, you'll see your add-ons on the no supported list.
I'll wait for your updates, thanks.

**Comment by madb1lly** [comment]
Hi @sv-sdeiac,

Thank you for managing to reproduce the result, however you haven't reproduced my steps, which I explained in the issue report. However, maybe you didn't understand that I was already using Fenix Nightly, not Fennec Nightly, so I updated the issue report to clarify this.

Unfortunately I can't reproduce your steps because I already have org.mozilla.fennec_aurora installed from the play store (for more than a year), I'm using it as my main browser and it has been a few weeks since it was updated to Fenix. If I follow your steps I need to uninstall my main browser and lose my session, which I don't want to do.

Anyway, to add more info it appears that I do actually have Privacy Badger installed, see #9145.

Cheers 🙂
